zetta_utils/convnet/architecture/convblock.py

```diff
@@ -9,13 +9,15 @@
 from typing_extensions import TypeAlias
 
 from zetta_utils import builder
+from zetta_utils.tensor_ops import crop_center
 from zetta_utils.typing import ensure_seq_of_seq
 
 Padding: TypeAlias = Union[Literal["same", "valid"], Sequence[int]]
 PaddingMode: TypeAlias = Literal["zeros", "reflect", "replicate", "circular"]
+ActivationMode: TypeAlias = Literal["pre", "post"]
 
 
-@builder.register("ConvBlock")
+@builder.register("ConvBlock", versions=">=0.0.1")
```

Comment:
We're trying to keep the versions tied to `zetta_utils` release version in the `pyproject.toml`.
Because it's a breaking change, we would need to bump the version to 0.0.2, and I think that's the version that the new ConvBlock should require.